django-s3-file-field-client/src/client.ts

```diff
@@ -1,5 +1,11 @@
 import axios from 'axios';
 
+export interface Progress {
+  readonly percentage: number;
```

Comment:
This value ranges `[0, 1]`, not `[0, 100]`, right? In that case, percentage is the wrong name. Maybe `fraction`, `proportion`, `ratio`... some other synonym?